src/NUnitEngine/nunit.engine/Internal/NUnitConfiguration.cs

```diff
@@ -59,35 +59,28 @@ public static string MonoExePath
         {
             get
```

Comment:
That's a fair point. I do have mono installed, but have neither the registry key nor mono on my path, so without artificially creating the registry key (which doesn't seem sensible!) - I'm not able to check this. Is it only certain versions of mono which install the key, do you know? If your installation does have the relevant key, would be great if you could take a look.

The tidying is non-essential to the PR, just habit! We could just remove the unused var instead, if we can't find a set up to verify this.